package.json

```diff
@@ -31,5 +33,9 @@
   "main": "server.js",
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
```

Comment:
Looks like you accidentally did an npm install in the root instead of the exercise directory. You should revert this file and package-lock.json.